xdmod/README.md

```diff
@@ -133,10 +157,989 @@ If you are installing the Job Performance module (as we are in this tutorial)
+---
+
+## Basic Configuration
+To begin the setup process for XDMoD you will want to type the following: 
+```shell
+[root@xdmod /] xdmod-setup
+```
+
+After which you should be greeted by the following screen: 
+```shell
+========================================================================
+Open XDMoD Setup
+========================================================================
+
+1) General Settings
+2) Database Settings
+3) Organization
+4) Resources
+5) Create Admin User
+6) Hierarchy
+7) Data Warehouse Batch Export
+8) Automatically Check for Updates
+9) SUPReMM
+q) Quit
+
+Select an option (1, 2, 3, 4, 5, 6, 7, 8, 9, q):
+```
+
+The next series of steps will involve completing each numbered setup item in turn from 1-8. We will complete the SUPReMM
+setup after we have successfully shredded, ingested, and aggregated the accounting data generated by the jobs we 
+submitted at the beginning of this tutorial.
+
+### General Settings [Documentation](https://open.xdmod.org/9.5/configuration.html#general-settings)
+
+```shell
+1) General Settings
+```
+
+- Press `1` followed by the `Enter` key to start the General Settings setup.
+
+```shell 
+Select an option (1, 2, 3, 4, 5, 6, 7, 8, 9, q): 1
+```
+
+#### Site Address
+
+The value you enter here should be the public facing URL that your XDMoD installation will be served from. For the 
+purpose of this tutorial we'll be using the default value.
+
+```shell
+========================================================================
+General Setup
+========================================================================
+
+The template Apache configuration file uses a virtual host
+listening on HTTPS port 443. The Site Address specified
+here should match the settings in the Apache configuration.
+
+Site Address: [https://localhost:4443/]
+```
+*Note: values contained in `[ ]` are default values that can be used by simply pressing the `Enter` key. These values have been modified specifically for this tutorial and may be different from the values you encounter while installing XDMoD at your home institution.*
+
+- Press the `Enter` key.
+
+#### Email Address
+This is the email address your XDMoD installation will use as the recipient of contact and user account requests.
+
+```shell
+The email address you specify will be used as the destination for any
+messages sent via the portal contact page as well as account requests.  In
+addition, any log messages configured for delivery via e-mail will be sent to
+this address.
+
+Email Address: [ccr-xdmod-help@buffalo.edu]
+```
+- Press the `Enter` key.
+
+#### Chromium Path
+XDMoD utilizes Chromium Headless to generate its images and reports, so the next step will be telling XDMoD where it can
+be found.
+
+```shell
+Chromium Path: [/usr/lib64/chromium-browser/headless_shell]
+```
+- Press the `Enter` key.
+
+#### Center Logo Path
+XDMoD does support per center branding, so if you do have an image you would like to use in branding your XDMoD 
```

Comment:
```suggestion
XDMoD supports a customizable logo image that is displayed in the web portal. If you have an image you would like to use in branding your XDMoD 
```

Reply:
In the past when we have had questions about branding, the topic of color comes up - and of course we don't easily support changing the color of the widgets

Reply:
*nod* Makes sense